doc/shader_compilation.md

```diff
@@ -2,15 +2,15 @@
 
 We use git branches to support shader compilation in the cloud. The `dev` branch contains only shader source files (in GLSL format), while the `main` branch contains generated shaders. On every push to the `dev` branch, a GitHub action runs which compiles the shaders and pushes those to `main`.
 
-Thus, you can run piet-gpu from the `main` branch without requiring any shader compilation tools. Also, the `dev` branch has a relatively clean history, and PRs can be made against it without having to worry about merge conflicts in the generated shader files.
+Thus, you can run vello from the `main` branch without requiring any shader compilation tools. Also, the `dev` branch has a relatively clean history, and PRs can be made against it without having to worry about merge conflicts in the generated shader files.
```

Comment:
I think most of this change makes sense, but this particular one is misleading, as it applies to the old piet-gpu before the Vello rename. I noticed this when writing my blog, and think maybe the best thing to do is delete this file (which is one reason I linked a particular SHA).

Reply:
Ok